### PR TITLE
[MO] - [logging] -> fix broken logging + support for oauth logging

### PR DESCRIPTION
**Pull request by see-quick**
This PR fixing the logging, which did not work for the clients. Moreover adding debug info about the config of each client.

Signed-off-by: morsak <xorsak02@stud.fit.vutbr.cz>

### CODE REVIEW

File: java/kafka/pom.xml [hunk 1]
```diff
@@ -16,7 +16,7 @@
     <properties>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
-        <log4j.version>2.13.3</log4j.version>
+        <log4j.version>2.7</log4j.version>
```
Comment (scholzj) [inline java/kafka/pom.xml]: Why are we downgrading to older version? I would not expect that is something we want to do.

**Review by Frawless — APPROVED** [review]
LGTM, but I agree with Jakub about log4j version

File: java/kafka/pom.xml (hunk 1 above)
Comment (see-quick) [reply java/kafka/pom.xml]: Hmmm, that was version, which was suggested by Marko but i can try it with the `2.13.3`

Comment (see-quick) [reply java/kafka/pom.xml]: Fixed

**Review by scholzj — APPROVED** [review]

**Review by ppatierno — APPROVED** [review]

**Comment by scholzj** [comment]
@see-quick Could you look at the conflict please?

**Comment by see-quick** [comment]
@scholzj Done :)

**Pull request merged**